pkg/packagekit/updates.scss

```diff
@@ -240,24 +240,6 @@ div.changelog {
     }
 }
 
-/* fixed width to avoid jumpy text between expanded/collapsed */
-.expander-caret {
-    width: 2ex;
-}
-
-.expander-title {
```

Comment:
I think this one positions the toggle a bit weirdly too much on the left:
current master:
![beforelogsupdate](https://user-images.githubusercontent.com/12330670/104915008-4c961900-5990-11eb-9f60-89309604b797.png)
in this PR:
![positionofalllogs](https://user-images.githubusercontent.com/12330670/104915018-515acd00-5990-11eb-90ef-b5802f83b98f.png)

Reply:
Hmm indeed, did not even realize it. Should I just move it in the middle or we need that divider CSS?

Reply:
So I removed that commit, since it's was indeed different visually component the one implemented than the one I replaced it with. I will rather make sure that it's PF3 independent in another pr.